examples/contracts.py

```diff
@@ -82,13 +83,18 @@ def track_cost(api: TokenApi, entity: Entity, message: str):
     print(message + "{} TOK".format(api.balance(entity) - balance_before))
 
 
+def _calc_digest(data):
+    hash_func = hashlib.sha256()
+    hash_func.update(data)
+    return hash_func.digest()
+
 def main():
     # create our first private key pair
-    entity1 = Entity()
+    entity1 = Entity(_calc_digest(str.encode('random1')))
```

Comment:
This doesn't look right @douglasb99 can you maybe explain what this issue is here?